src/main/java/edu/greenblitz/pegasus/RobotMap.java

```diff
@@ -89,6 +90,9 @@ public static class PressureSensor {
 			}
 		}
 		
+		public static class DigitalInputMap {
+			public static final int MACRO_SWITCH = 0;
```

Comment:
consider changing the name to somthing more informative? myabe "MACRO_SWITCH_PORT"
```suggestion
			public static final int MACRO_SWITCH = 0;
```